### PR TITLE
Update easymde to 2.10.1

### PR DESCRIPTION
**Pull request by kolaente**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
   "repository": "github:NikulinIlya/vue-easymde",
   "license": "MIT",
   "dependencies": {
-    "easymde": "^2.8.0",
+    "easymde": "^2.10.0",
     "marked": "^0.7.0"
```
Comment (kolaente) [inline package.json:29]: Easymde itself requires `marked`, so I'm not sure if this is needed here. Is there a special reason to include this?

**Comment by spech66** [comment]
Could you please update this to `2.10.1` and create a new release?

easymde brings it's own marked dependency https://www.npmjs.com/package/easymde

**Comment by kolaente** [comment]
@spech66 Done!

**Comment by spech66** [comment]
@kolaente Thank you! Now we would need a release :see_no_evil:

**Comment by kolaente** [comment]
@NikulinIlya Any thoughts on merging this?

**Comment by kolaente** [comment]
Thanks!

**Pull request merged**